### PR TITLE
Updating install for 3.2

### PR DESCRIPTION
**Pull request by bobdenotter**
Bolt 3.2 specific changes for the impending release.

### CODE REVIEW

File: docs/configuration/introduction.md [hunk 1]
```diff
@@ -4,20 +4,34 @@ title: Setting up Bolt
 Setting up Bolt
 ===============
 
-By default, Bolt is configured to use an SQLite database. See [configure the database](database),
-if you want to change this to either MySQL or PostgreSQL.
+To get started with setting up the first Admin user in your new Bolt install,
+open the Bolt site in your browser. You should be greeted by the screen to
+create the first user. If you see this 'Create the first user'-screen, do so
```
Comment (GawainLynch) [inline docs/configuration/introduction.md]: Hyphen not needed after quotation

File: docs/extensions/advanced/fieldtypes.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Providing New Field Types
 Extensions: Providing a New Field Type
 ======================================
 
-Bolt 3.0 now gives extension developers access to the complete lifecycle of a
+Bolt 3 now gives extension developers access to the complete lifecycle of a
```
Comment (PhillippOhlandt) [inline docs/extensions/advanced/fieldtypes.md]: Shouldn't only go into 3.2

File: docs/configuration/introduction.md (hunk 1 above)
Comment (bobdenotter) [inline docs/configuration/introduction.md]: Ninja-fixed.

**Comment by GawainLynch** [comment]
Merged … on top of #522 into `release/3.0` via 4940c923d699b9a6c8b398e59fe161d350e425e8

**Pull request closed without merging**